src/Traits/JsonAwareTrait.php

```diff
@@ -2,9 +2,11 @@
 
 namespace Equip\Data\Traits;
 
-trait JsonAwareTrait
+trait JsonAwareTrait /* implements JsonSerializable */
```

Comment:
I wish traits could implement interfaces. I like knowing how the trait should be used and this is the best I've come up with.